CMSgov/easi-app
# [NOREF] Send LCID alerts to requester

# NOREF

## Changes and Description

- Use `listAllRecipients` to send to requester AND ITGovernance mailbox

## How to test this change

- `scripts/dev up` to start the app
- Create an IT Gov request
- Schedule an LCID that expires in 50 days
- Restart the app to re-trigger the alert check
- Check http://localhost:1080 to ensure the email is sent to the requester and the IT Governance mailbox

![image](https://github.com/CMSgov/easi-app/assets/15203744/ab02d351-42dc-47c1-a5e6-64086b3fec53)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
